src/platform/datapath_raw_socket.c

```diff
@@ -252,27 +263,10 @@ CxPlatDpRawParseIPv6(
     Packet->Tuple->LocalAddress.Ipv6.sin6_family = AF_INET6;
     CxPlatCopyMemory(&Packet->Tuple->LocalAddress.Ipv6.sin6_addr, IP->Destination, sizeof(IP->Destination));
 
-    if (IP->NextHeader == 17) {
-        CxPlatDpRawParseUdp(Datapath, Packet, (UDP_HEADER*)IP->Data, Length);
-    } else if (IP->NextHeader != 59) {
-        const uint8_t* Data = IP->Data;
-        do {
-            if (Length < sizeof(IPV6_EXTENSION)) {
-                return;
-            }
-            const IPV6_EXTENSION* Extension = (const IPV6_EXTENSION*)Data;
-            const uint16_t ExtLength = sizeof(IPV6_EXTENSION) + Extension->Length * sizeof(IPV6_EXTENSION);
-            if (Length < ExtLength) {
-                return;
-            }
-            Length -= ExtLength;
-            Data += ExtLength;
-            if (Extension->NextHeader == 17) {
-                CxPlatDpRawParseUdp(Datapath, Packet, (UDP_HEADER*)Extension->Data, Length);
-            } else if (Extension->NextHeader == 59) {
-                return;
-            }
-        } while (TRUE);
+    if (IP->NextHeader == IPPROTO_UDP) {
```

Comment:
The reason I had all this "crap" before was because I noticed real IPv6 traffic had a lot of extensions before UDP in most cases. If you want to remove it for now, I'm fine with that, but I suspect it will need to come back.

Reply:
Are you sure in most cases we will have extensions? I just took a packet capture for buffering a youtube video and I can't find any extensions in the QUIC flow.

Reply:
Maybe it was just the local traffic in the lab. We can keep this as is for now.